gemini/js-web-genkit/server.ts

```diff
@@ -1,35 +1,35 @@
-import { gemini15Flash, googleAI } from "@genkit-ai/googleai";
+import { googleAI } from "@genkit-ai/googleai";
+import express from "express";
 import * as fs from 'fs/promises';
 import { genkit, z } from 'genkit';
-import { logger } from 'genkit/logging'
-import express from "express";
+import { logger } from 'genkit/logging';
 
 const port = process.env.PORT || 3000;
 
 export const ai = genkit({
-    plugins: [googleAI()],
+  plugins: [googleAI()],
 });
 
-logger.setLogLevel('debug')
-const recipiePrompt = ai.definePrompt({
-        name: 'Recipies prompt',
-        model: gemini15Flash,
-        input: {
-            schema: z.object({
-                photoUrl: z.string(),
-                userPrompt: z.string(),
-            })
-        },
-        output: {
-            format: 'json',
-            schema: z.object({
-                recipe: z.string()
-                    .describe("A recipe, starting with a title, in markdown format"),
-                tags: z.array(z.string())
-                    .describe("Two to Four 1-word keyword tags for the recipe, lowercase only"),
-            })
-        },
-    },`
+logger.setLogLevel('debug');
+const recipePrompt = ai.definePrompt({
```

Comment:
nit: typo

Reply:
hm im not seeing it... `recipe`?

Reply:
oh i think it was on the next line ... "Recipies prompt". fixed!